README.md

```diff
@@ -83,19 +83,20 @@ using an SSL/TLS certificate. The latter is utterly important, as there will
 be person-related information transferred on this application that you may
 not want everybody to see).
 
-Deployment with Docker
-----------------------
+Deployment with Docker Compose
+------------------------------
 
-With Docker, you can just run the following commands to get started:
+With Docker Compose, you can just run the following commands to get started:
 
-    docker build .
-    docker run
+    docker-compose up
 
-Maybe in the future, we will also provide "official" modelreg docker images,
-so you won't have to build it yourself.
+We will also provide "official" modelreg docker images which you can find [on Docker Hub](https://hub.docker.com/r/modelreg/modelreg/).
+
+To get the image up and running, you need to copy `.env.example` to `.env` and
+configure this file to your liking.
```

Comment:
We should change the env variable configuration below as well, as we're not suggesting the user to use "plain" docker

Reply:
Yes, this is correct.

Reply:
OK as discussed offline, we'll do that in a separate PR